app/views/naming/_form.html.erb

```diff
@@ -33,8 +33,8 @@
 
     <div class="form-group">
       <%= label_tag(:vote_value, :form_naming_confidence.t + ":") %><br>
-      <%= select(:vote, :value, Vote.confidence_menu,
-                 {include_blank: action == :create}, class: "form-control") %>
+      <%= select(:vote, :value, options_for_select(Vote.confidence_menu, vote.value),
```

Comment:
When editing an naming the current vote value is selected, unless there is no vote value in which the user will need to select a vote value.